chainer/functions/connection/deconvolution_nd.py

```diff
@@ -326,54 +326,99 @@ def deconvolution_nd(x, W, b=None, stride=1, pad=0, outsize=None,
+      channels, respectively.
+    - :math:`d_1, d_2, ..., d_N` are the size of each axis of the input's
+      spatial dimensions, respectively.
+    - :math:`k_1, k_2, ..., k_N` are the size of each axis of the filters,
+      respectively.
+    - :math:`l_1, l_2, ..., l_N` are the size of each axis of the output's
+      spatial dimensions, respectively.
+    - :math:`p_1, p_2, ..., p_N` are the size of each axis of the spatial
+      padding size, respectively.
+
+    Let :math:`(s_1, s_2, ..., s_N)` be the stride of filter application.
+    Then, the output size :math:`(l_1, l_2, ..., l_N)` is determined by the
+    following equations:
+
+    .. math::
+
+       l_n = s_n (d_n - 1)  + k_n - 2 p_n \ \ (n = 1, ..., N)
 
     Args:
-        x (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Input data of shape :math:`(n, c_I, d_1, d_2, ..., d_N)`.
-        W (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Weight data of shape :math:`(c_I, c_O, k_1, k_2, ..., k_N)`.
-        b (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Bias vector of length :math:`c_O` (optional).
-        stride (int or tuple of ints): Stride of filter applications
-            :math:`(s_1, s_2, ..., s_N)`. ``stride=s`` is equivalent to
-            ``(s, s, ..., s)``.
-        pad (int or tuple of ints): Spatial padding size for input arrays
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable of shape :math:`(n, c_I, d_1, d_2, ..., d_N)`.
+        W (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Weight variable of shape :math:`(c_I, c_O, k_1, k_2, ..., k_N)`.
+        b (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            One-dimensional bias variable with length :math:`c_O` (optional).
+        stride (:class:`int` or :class:`tuple` of :class:`int` s):
+            Stride of filter applications :math:`(s_1, s_2, ..., s_N)`.
+            ``stride=s`` is equivalent to ``(s, s, ..., s)``.
+        pad (:class:`int` or :class:`tuple` of :class:`int` s):
+            Spatial padding width for input arrays
             :math:`(p_1, p_2, ..., p_N)`. ``pad=p`` is equivalent to
             ``(p, p, ..., p)``.
-        outsize (tuple of ints): Expected output size of deconvolutional
-            operation. It should be a tuple of ints
-            :math:`(out_1, out_2, ..., out_N)`. Default value is ``None`` and
-            the outsize is estimated by input size, stride and pad.
+        outsize (:class:`tuple` of :class:`int` s):
+            Expected output size of deconvolutional operation. It should be a
+            tuple of ints :math:`(l_1, l_2, ..., l_N)`. Default value is
+            ``None`` and the outsize is estimated by input size, stride and
+            pad.
```

Comment:
User-specified outsize should satisfy `get_conv_outsize(outsize) == insize`, that is checked [here](https://github.com/pfnet/chainer/blob/master/chainer/functions/connection/deconvolution_nd.py#L55).